test/integration/join-test.js

```diff
@@ -108,6 +108,22 @@ testRingpopCluster({
 });
 
 testRingpopCluster({
+    tap: function tap(cluster) {
+        cluster.forEach(function eachNode(node) {
+            node.gossip.start = function noop() {};
+        });
+    },
+    waitForConvergence: false
+}, 'do not disseminate join list', function t(bootRes, cluster, assert) {
+    cluster.forEach(function eachNode(node) {
```

Comment:
call `assert.plan(X)` to make sure that the "right" number of assertions are happening. I suggest this because for some reason if cluster were empty, forEach wouldnt iterate over anything and the test would still pass.